centralledger/Chart.yaml

```diff
@@ -14,33 +14,35 @@ maintainers:
     email: miguel.debarros@modusbox.com
   - name: Tom Daly
     email: tomd@crosslaketech.com
+  - name: Steven Oderayi
+    email: steven.oderayi@infitx.com
 dependencies:
   - name: centralledger-service
-    version: 14.0.0
+    version: 14.1.0
     repository: "file://./chart-service"
     condition: centralledger-service.enabled
   - name: centralledger-handler-transfer-prepare
-    version: 14.0.0
+    version: 14.1.0
     repository: "file://./chart-handler-transfer-prepare"
     condition: centralledger-handler-transfer-prepare.enabled
   - name: centralledger-handler-transfer-position
-    version: 14.0.0
+    version: 14.1.0
     repository: "file://./chart-handler-transfer-position"
     condition: centralledger-handler-transfer-position.enabled
   - name: centralledger-handler-transfer-get
-    version: 14.0.0
+    version: 14.1.0
     repository: "file://./chart-handler-transfer-get"
     condition: centralledger-handler-transfer-get.enabled
   - name: centralledger-handler-transfer-fulfil
-    version: 14.0.0
+    version: 14.1.0
     repository: "file://./chart-handler-transfer-fulfil"
     condition: centralledger-handler-transfer-fulfil.enabled
   - name: centralledger-handler-timeout
-    version: 14.0.0
+    version: 14.1.0
     repository: "file://./chart-handler-timeout"
     condition: centralledger-handler-timeout.enabled
   - name: centralledger-handler-admin-transfer
-    version: 14.0.0
+    version: 14.2.0
```

Comment:
may have to change based on centralledger/chart-handler-admin-transfer/Chart.yaml comment

Reply:
@elnyry-sam-k Minor version bump here seem okay since we bumped the minor version in the appVersion.

Reply:
minor version is fine but wanted to know if there's a reason to skip v14.1.0 and go from x.0.0 to x.2.0